src/util/prolog/prolog.test.ts

```diff
@@ -0,0 +1,296 @@
+
+import { PrologSession } from "./prologSession";
+import * as program from "./testPrograms";
+
+// Expected results are consistent with SWI-Prolog behavior except that they usually have an extra "false."
```

Comment:
What does the `"false"` mean? Why is it there? I didn't quite understand that from the PR description either.